docs/user-guide/scaling.rst

```diff
@@ -0,0 +1,319 @@
+.. _MongoDB: http://www.mongodb.org/
+.. _Apache Qpid: https://qpid.apache.org/
+.. _RabbitMQ: http://www.rabbitmq.com/
+.. _MongoDB Deployment: http://www.mongodb.org/about/introduction/#deployment-architectures
+.. _Apache Qpid HA docs: https://qpid.apache.org/releases/qpid-0.28/cpp-broker/book/chapter-ha.html
+.. _RabbitMQ HA docs: http://www.rabbitmq.com/ha.html
+.. _mod_status: https://httpd.apache.org/docs/2.2/mod/mod_status.html
+.. _HAProxy: http://www.haproxy.org/
+
+Scaling Pulp
+============
+
+Great effort has been put into Pulp to make it scalable. A default Pulp
+install is an "all-in-one" style setup with everything running on one machine.
+However, Pulp supports a clustered configuration with different servers or
```

Comment:
I'd change "configuration with different servers or" to "deployment across multiple machines and/or"